compiler-plugin/src/test/kotlin/arrow/meta/plugins/comprehensions/ComprehensionsTest.kt

```diff
@@ -310,8 +254,8 @@ class ComprehensionsTest {
         |
         |""".source
       },
-      assert = {
-        failsWith { it.contains("Unresolved reference: a") }
+      asserts = {
+        listOf(failsWith { it.contains("Unresolved reference: a") })
```

Comment:
💡 Something used in some Android testing frameworks when it comes to multiple matches / assertions is `allOf()` which can be just a forwarding to listOf. Just a random idea. lisOf is also fine!

Reply:
Thanks @JorgeCastilloPrz! It seems like `assertAll` from JUnit5. Thanks for the idea, added!